app-creativity/rnote/autobuild/defines

```diff
@@ -0,0 +1,7 @@
+PKGNAME=rnote
+PKGDES="Sketch and take handwritten notes."
```

Comment:
Please make this more descriptive: what is it and what does it do? You have answered the latter but not the former.

Also, on styling: do not include a period point (.) at the end.